manzif/Banka
# #165821654 add graphs for admin

### What does this PR do?
Add graphs to admin

### Description of Task to be completed?
As an admin, I should be able to see graphs representing how many users we are getting per a certain period of time.
Change authentication for login, from now on we need an email or username instead of full names.
### How should this be manually tested?
If you visit banka UI, you will see that it requires you to login with username or email. And if you are an admin, the first pages of account is user graphs representation

What are the relevant pivotal tracker stories?
[#165821654](https://www.pivotaltracker.com/story/show/165821654)
